src/utils/tool-check.ts

```diff
@@ -21,7 +21,7 @@ export function tryPromptForUpdatingTool(version: semver.SemVer | null) {
 export async function getToolVersion(
   executable: string
 ): Promise<semver.SemVer | null> {
-  const { stdout } = execa.sync(executable, ["-V"], { timeout: 5000 });
+  const { stdout } = await execa(executable, ["-V"], { timeout: 5000 });
```

Comment:
Hey this fixed a bug, believe you or not:

https://github.com/vscode-shellcheck/vscode-shellcheck/pull/180

Reply:
Here: https://github.com/vscode-shellcheck/vscode-shellcheck/pull/166

Reply:
Ooops...